wgpu-rs/src/lib.rs

```diff
@@ -187,6 +213,37 @@ impl Device {
         }
     }
 
+    pub fn create_bind_group(&self, desc: &BindGroupDescriptor) -> BindGroup {
+        use BindingResource::*;
```

Comment:
let's not wildcard include

Reply:
I think, it's useful for enums, but I can also call all the enums explicitly.